### PR TITLE
Fix add to scriptFileNameSet path format

### PR DESCRIPTION
**Pull request by yoyo930021**
Fixed #1450 #1449

### CODE REVIEW

**Comment by yoyo930021** [comment]
@octref
Sorry I have an error fixed in #1450.
Path format was error in Typescript Program when using windows.

This PR can fix it.

File: server/src/services/typescriptService/serviceHost.ts [hunk 1]
```diff
@@ -289,7 +289,7 @@ export function getServiceHost(
                 TextDocument.create(uri.toString(), 'vue', 0, tsModule.sys.readFile(resolvedFileName) || '')
               );
               localScriptRegionDocuments.set(resolvedFileName, doc);
-              scriptFileNameSet.add(resolvedFileName);
+              scriptFileNameSet.add(uri.path);
```
Comment (octref) [inline server/src/services/typescriptService/serviceHost.ts]: Same as above

File: server/src/services/typescriptService/serviceHost.ts [hunk 2]
```diff
@@ -219,7 +219,7 @@ export function getServiceHost(
               TextDocument.create(uri.toString(), 'vue', 0, tsModule.sys.readFile(fileName) || '')
             );
             localScriptRegionDocuments.set(fileName, doc);
-            scriptFileNameSet.add(fileName);
+            scriptFileNameSet.add(uri.path);
```
Comment (octref) [inline server/src/services/typescriptService/serviceHost.ts]: You can use getFilePath from `paths.ts` util.

**Comment by octref** [comment]
Thanks, there are some documentations in `paths.ts` that you can read to deal with Windows path handling.

File: server/src/services/typescriptService/serviceHost.ts (hunk 1 above)
Comment (yoyo930021) [reply server/src/services/typescriptService/serviceHost.ts]: localScriptRegionDocuments set uri.fsPath
scriptFileNameSet set resolvedFileName
Path format is be right.

File: server/src/services/typescriptService/serviceHost.ts (hunk 2 above)
Comment (yoyo930021) [reply server/src/services/typescriptService/serviceHost.ts]: I changed the method.

**Comment by octref** [comment]
I made the path naming consistent and left some more notes in `paths.ts`. I tested and this works now on Windows.

**Pull request merged**